app/components/posting-list.js

```diff
@@ -10,7 +10,7 @@ export default class PostingList extends React.PureComponent {
     }
 
     renderPosting(row) {
-        return <Posting addLike={this.props.addLike} {...(row.item)} />;
+        return <Posting addLike={this.props.addLike} {...(Object.assign(this.props.navigation, row.item))} />;
```

Comment:
isn't it basically 
```suggestion
        return <Posting addLike={this.props.addLike} { ...this.props.navigation, ...row.item } />;
```

Reply:
didn't work... as it merges with eventually existing props from React.PureComponent I assume